content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,117 @@
 ---
 title: "WebRTC"
-description: "WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p."
+description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other nodes."
 weight: 110
 aliases:
     - "/concepts/transports/webrtc"
 ---
 
-Coming soon!
+## What is WebRTC?
+
+[WebRTC (Web Real-Time Communications)](https://webrtc.org/) is a framework for real-time
+communication and is used to establish browser-to-server and browser-to-browser connections
+between applications. WebRTC serves as a good choice for applications that need built-in support
```

Comment:
Can we say something like "WebRTC was developed by ... to facilitate real-time communication such as audio and video conferencing".